model/map-hot-rod/src/main/java/org/keycloak/models/map/storage/hotRod/group/HotRodGroupEntity.java

```diff
@@ -109,6 +109,7 @@ public void setName(String name) {
     @ProtoField(number = 6)
     public String parentId;
 
+    @ProtoDoc("@Field(index = Index.YES, store = Store.YES)")
     @ProtoField(number = 7)
     public Set<HotRodAttributeEntityNonIndexed> attributes;
```

Comment:
We need to also change the type of the attribute.

```suggestion
    public Set<HotRodAttributeEntity> attributes;
```

Reply:
Addressed this in latest push